src/main/java/com/faforever/client/achievements/AchievementItemController.java

```diff
@@ -87,7 +87,7 @@ public void setPlayerAchievement(PlayerAchievement playerAchievement) {
       imageView.setEffect(null);
     }
 
-    if (AchievementType.INCREMENTAL == achievementDefinition.getType()) {
+    if (AchievementType.INCREMENTAL.equals(achievementDefinition.getType())) {
```

Comment:
To compare enums, we always use `==`

See https://stackoverflow.com/a/2937561/513451

Reply:
cool explaination